dietpi/dietpi-software

```diff
@@ -3301,7 +3301,7 @@ _EOF_
 			[[ $target ]] && target="-d $target"
 			l_message='Unzipping archive' G_RUN_CMD unzip -o $file $target
 
-		elif [[ $type == tar ]]; then # .bz2 .gz may require different extraction vars if I remember correctly?
```

Comment:
@Fourdee 
I tested it on all so far affected installs: `tar` uses `gzip`/`bzip2` module very well by itself. The additional arguments are needed only for packing, not for unpacking.